18.0 t 61746 jjs

jjscarafia opened the pull request.

jjscarafia committed:
[IMP] _payment_pro: better ux and handling of counterpart currency

Mejoramos
1. el dominio de la counterpart, no tiene sentido elegir moneda de la compañía
2. reseteamos la moneda seleccionada al cambiar diario
3. dejamos el campo visible tmb cuando los pagos son en un diario con divisa (por ej. puedo cobrar "USD" pero decir qu estoy cancelando "EUR")

jjscarafia committed:
[IMP] _payment_pro: better amount_company_currency views
1. Simplificamos código
2. hacemos que el rate aparezca en misma línea

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/854.png)](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/854)

jjscarafia committed:
[REM] pay_pro: remove onchange to clean counterpart currency

1. para el onchange que se implementó acá https://github.com/ingadhoc/account-payment/pull/849/changes
2. también se había implementado solución en paralelo convirtiendo el campo a computado.

Mantenemos approach de computado por:
* es más robusto (funciona por api, expo, impo. es el standard salvo para algún caso puntual donde se necesite onchange)
* además el método implementado en el compute resetea el valor solo cuando es necesario

jjscarafia commented:
@roboadhoc r+

roboadhoc commented:
@jjscarafia because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

jjscarafia commented:
@roboadhoc rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
